.github/workflows/nightly-bnb.yml

```diff
@@ -25,7 +25,7 @@ jobs:
       TEST_TYPE: "single_gpu_${{ matrix.docker-image-name }}"
     container:
       image: ${{ matrix.docker-image-name }}
-      options: --gpus all --shm-size "16gb" --ipc host -v /mnt/cache/.cache/huggingface:/mnt/cache/ -e NVIDIA_DISABLE_REQUIRE=true
+      options: --gpus all --shm-size "16gb" --ipc host -v /mnt/cache/.cache/huggingface:/mnt/cache/
```

Comment:
the env variable `NVIDIA_DISABLE_REQUIRE` is set on the very top so these are not needed